docs/content/3.application-side/5.guest-mode.md

```diff
@@ -15,6 +15,7 @@ This behavior is useful for login pages that you don't want to be visitable by l
 Briefly summarized, you can enable guest mode on a page by passing the following configuration:
 ```ts
 definePageMeta({
+  middleware: 'auth',
```

Comment:
Good addition - however this is only the case if you've also disabled the global middleware!

(at least that's how it should be, if it isn't, that's the real bug!)

So please change this example to use the `::code-group` element and give one example with and one example without, name them accordingly. E.g.:
- With global middleware enabled
- With global middleware disabled

Or something like that.

Thanks ❤️

Reply:
Ok, I don't use the global middleware so didn't now that. I will change it.

Reply:
very legit and it's a great improvement to the docs! (: